src/main/java/ch/njol/skript/sections/EffSecSpawn.java

```diff
@@ -143,6 +142,7 @@ protected TriggerItem walk(Event e) {
 				// Copy the local variables from the calling code to this section
 				Variables.setLocalVariables(spawnEvent, localVars);
 				trigger.execute(spawnEvent);
+				Variables.setLocalVariables(e, localVars); // Carry over variable changes to the rest of the main trigger
```

Comment:
Will this carry over the new local variables in that spawn section? e.g.
```v
set {_v1} to "Test"
spawn cow at player:
    broadcast {_v1} # Should broadcast "Test"
    set {_v1} to "UPDATED"
    set {_v2} to "Test2" # new local variable here that should be carried over outside
    broadcast {_v1} # Should broadcast "UPDATED"
    broadcast {_v2} # Should broadcast "Test2"

broadcast {_v1} # Should broadcast "UPDATED"
broadcast {_v2} # Should broadcast "Test2"
```

Reply:
Yep!